src/olympia/files/migrations/0029_auto_20230301_1843.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 3.2.18 on 2023-03-01 18:43
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('files', '0028_set_approval_date'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
```

Comment:
`makemigrations` added this, even though my patch does not include this change. I wasn't sure if I should just keep it, because it's probably something that is missing from another migration, or if I should remove it from this file, as I didn't make this change.

Reply:
Well, that broke everything on CI, so I removed it. Not sure why `makemigrations` wanted to do that.

Reply:
It's a dependency issue - migration `0035` in `versions` uses that field, so if you want to remove it, you need to add it as dependency of your migration to force the migration engine to execute migrations in an order that isn't broken.

We are usually deliberately not removing the fields right away from the database for backwards-compatility with existing code when we deploy, but in this case it was also useful to still have the database data laying around to backfill some more data (in fact that's what migration `0035` in `versions` did) but it's probably ok to remove now.